src/redisai.h

```diff
@@ -93,15 +93,15 @@ int MODULE_API_FUNC(RedisAI_ModelRunCtxAddInput)(RAI_ModelRunCtx *mctx, const ch
 int MODULE_API_FUNC(RedisAI_ModelRunCtxAddOutput)(RAI_ModelRunCtx *mctx, const char *outputName);
 size_t MODULE_API_FUNC(RedisAI_ModelRunCtxNumOutputs)(RAI_ModelRunCtx *mctx);
 RAI_Tensor *MODULE_API_FUNC(RedisAI_ModelRunCtxOutputTensor)(RAI_ModelRunCtx *mctx, size_t index);
-void MODULE_API_FUNC(RedisAI_ModelRunCtxFree)(RAI_ModelRunCtx *mctx);
+void MODULE_API_FUNC(RedisAI_ModelRunCtxFree)(RAI_ModelRunCtx *mctx, bool freeTensors);
```

Comment:
breaking change
also
why do you need such a variant? we have `shallow copy` for tensors, so freeing them will just decrease their counter.

Reply:
You're right, I don't know why this variant is needed. I'll ask Luca (I guess that he changed the free function without thinking about the API)